### PR TITLE
Add the solana-wallet documentation

### PR DESCRIPTION
**Pull request by garious**
#### Problem

There doesn't seem to be a way to publish bin docs to crates.io.

#### Summary of Changes

* Publish solana-wallet documentation via the markdown book

### CODE REVIEW

**Comment by garious** [comment]
@CriesofCarrots, can you take a pass through these examples and comment on commands that either don't exist, missing commands, or output different text than the RFC described?

**Comment by CriesofCarrots** [comment]
Ha, comments passed each other in the ether. I will do so!

File: src/wallet.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+The [solana crate](https://crates.io/crates/solana) is distributed with a command-line interface tool
+
+Usage:
+
+```sh
+$ solana-wallet [common-options] [command] [command-specific options]
+```
+
+`common-options` include:
+* `--fee xyz` - Transaction fee (0 by default)
```
Comment (CriesofCarrots) [inline src/wallet.md]: Current common-options:
- network
- keypair path
- gossip timeout
- rpc-port
- proxy (to use TLS for rpc)

File: src/wallet.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+* `--output file` - Write the raw Transaction to a file instead of sending it
+
+`command` variants:
+* `pay`
+* `cancel`
+* `send-signature`
+* `send-timestamp`
+
```
Comment (CriesofCarrots) [inline src/wallet.md]: Additional available commands:
- address
- airdrop
- balance
- confirm
- deploy
- get-transaction-count

Comment (CriesofCarrots) [reply src/wallet.md]:
```suggestion
#### Get Pubkey
```sh
// Command
$ solana-wallet address

// Return
<PUBKEY>
```
```

Comment (CriesofCarrots) [reply src/wallet.md]:
```suggestion
#### Airdrop Tokens
```sh
// Command
$ solana-wallet airdrop 123

// Return
"Your balance is: 123"
```

Comment (CriesofCarrots) [reply src/wallet.md]:
```suggestion
#### Get Balance
```sh
// Command
$ solana-wallet balance

// Return
"Your balance is: 123"
```

Comment (CriesofCarrots) [reply src/wallet.md]:
```suggestion
#### Confirm Transaction
```sh
// Command
$ solana-wallet confirm <TX_SIGNATURE>

// Return
"Confirmed" / "Not found"
```

Comment (CriesofCarrots) [reply src/wallet.md]:
```suggestion
#### Deploy program
```sh
// Command
$ solana-wallet deploy <PATH>
// Return
<PROGRAM_ID>
```

Comment (CriesofCarrots) [reply src/wallet.md]:
```suggestion
#### Get Transaction Count
```sh
// Command
$ solana-wallet get-transaction-count

// Return
"123"
```

**Pull request merged**